### ISSUE TITLE
Cookbook name should be newrelic_ng

### ISSUE THREAD

**Issue by chr4** [chr4-cookbooks/newrelic-ng#28]
Convenction is afaik that cookbooks should prefer `_` over `-`. This might be changed in a future version.

**Comment by ofir-petrushka** [comment]
Doing this would be a breaking change, assuming attributes are going to get renamed...
Is it worth it?.. Will anyone gain anything from this? is the gain worth the pain for all users updating their cookbooks/recipes?..

**Comment by chr4** [comment]
Yeah, this is not going to happen soon - Maybe when releasing a next major release with breaking changes.

**Comment by jeffbyrnes** [comment]
I’d say it’s alright, considering `build-essential` is still named with a hyphen. It _is_ best practice to use underscores nowadays, but I think you can leave it alone & avoid the serious pain this would bring.
